.github/workflows/publish-rc.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Publish Release Candidate'
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-rc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+      - uses: actions/checkout@v2
+        run: echo NVMRC=`cat .nvmrc` >> $GITHUB_ENV
+
+      - name: Install dependencies and publish
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NVMRC }}
+        run: yarn && yarn publish:next
+
+      # https://github.community/t/how-to-push-to-protected-branches-in-a-github-action/16101/30
+      - name: Branch protection OFF
+        uses: octokit/request-action@v2.x
+        with:
+          route: PUT /repos/:repository/branches/main/protection
+          repository: ${{ github.repository }}
+          required_status_checks: |
+            null
+          enforce_admins: |
+            null
+          required_pull_request_reviews: |
+            null
+          restrictions: |
+            null
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Commit release
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: 'Publish release candidate'
+
+      # https://github.community/t/how-to-push-to-protected-branches-in-a-github-action/16101/30
+      - name: Branch protection ON
+        uses: octokit/request-action@v2.x
+        with:
+          route: PUT /repos/:repository/branches/main/protection
+          repository: ${{ github.repository }}
+          # https://docs.github.com/en/rest/reference/repos#update-branch-protection-preview-notices
+          mediaType: |
+            previews:
+              - luke-cage
+          required_status_checks: |
+            strict: false
```

Comment:
`strict` determines this option
![image](https://user-images.githubusercontent.com/175330/132251881-7de0948c-84d8-48ca-b96b-4a07a981da28.png)